src/encoded/docs/extended_description_StructuralVariantSample_inheritance_mode.html

```diff
@@ -28,3 +28,11 @@ <h5>
 <li> Variants with genotypes that do not meet any of the high relevance criteria above are not assigned inheritance mode labels.
 </ul>
 </p>
+
+<p><b>Ambiguous</b>: </p>
+
+<p>
+  If a sample was submitted to CGAP with the sex indicated as "U" or "Unknown", then autosome inheritance modes and genotypes
+  will still be calculated normally but sex chromosome inheritance mode and genotype cannot be properly determined. These will
+  be marked ambiguous.
```

Comment:
For reasons unknown to me, paragraphs aren't rendered intuitively from these files as new lines are not separated with a space. Everything has to be on one line or an explicit space (`&nbsp;`) has to be inserted at the beginning of the new line in order to achieve the spacing desired.

Also, consider style changes as below. 
```suggestion
  If a sample is submitted to CGAP with the sex indicated as "U" or "Unknown", then autosomal inheritance modes and genotypes are calculated normally but sex-linked inheritance modes and genotypes cannot be properly determined. These will be marked "Ambiguous."
```